charts/gha-runner-scale-set-controller/templates/NOTES.txt

```diff
@@ -1,3 +1,5 @@
 Thank you for installing {{ .Chart.Name }}.
 
 Your release is named {{ .Release.Name }}.
+
+WARNING: value specified under `image.pullPolicy` will no longer be applied to the listener pod spec in the next release. Please use the `listenerTemplate` in the gha-runner-scale-set chart to control the image pull policy of the listener.
```

Comment:
Awesome, I would also suggest that we create an announcement: https://github.com/actions/actions-runner-controller/discussions/categories/announcements with:
- what's being deprecated and why
- timelines
- apply the [deprecation](https://github.com/actions/actions-runner-controller/labels/deprecation) and [breaking](https://github.com/actions/actions-runner-controller/labels/breaking) labels on it